source/beamline/PGM/PGMBeamline.cpp

```diff
@@ -131,6 +131,34 @@ void PGMBeamline::setupMono()
 
 void PGMBeamline::setupComponents()
 {
+	// ==== set up the beamline status
+	// Photon Shutters
+	AMControl *photonShutter1 = new AMReadOnlyPVControl("PhotonShutter1", "PSH1411-I00-01:state", this, "Photon Shutter 1");
+	AMControl *photonShutter2 = new CLSExclusiveStatesControl("PhotonShutter2", "PSH1411-I00-02:state", "PSH1411-I00-02:opr:open", "PSH1411-I00-02:opr:close", this, "Photon Shutter 2");
+	AMControl *photonShutter3 = new CLSExclusiveStatesControl("PhotonShutter3", "PSH1611-3-I20-01:state", "PSH1611-3-I20-01:opr:open", "PSH1611-3-I20-01:opr:close", this, "Photon Shutter 3");
+	AMControl *safetyShutter1 = new CLSExclusiveStatesControl("SafetyShutter1", "SSH1411-I00-01:state",  "SSH1411-I00-01:opr:open", "SSH1411-I00-01:opr:close", this, "Safety Shutter 1");
+
+	beamlineShutters_ = new CLSShutters(QString("PGM Shutters"), this);
```

Comment:
@helfrij Yes, the beam on/off order are different from beamline to beamline. This CLSShutters/CLSValves/CLSBeamlineStatus classes might be refactored when working on #2355, which will try to create a generic ways to figure out the order.